packages/ckeditor5-package-generator/lib/utils/choose-package-manager.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license Copyright (c) 2020-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { prompt } = require( 'inquirer' );
+const isYarnInstalled = require( './is-yarn-installed' );
+
+const NPM = 'npm';
+const YARN = 'yarn';
+
+/**
+ * @typedef {Object} ChoosePackageManagerArgs
+ * @property {Boolean} useNpm
+ * @property {Boolean} useYarn
+ */
+
+/**
+ * @param {ChoosePackageManagerArgs} args
+ * @returns {NPM|YARN}
+ */
+module.exports = async function choosePackageManager( { useNpm = false, useYarn = false } = {} ) {
```

Comment:
Default values should come from the `packages/ckeditor5-package-generator/bin/index.js`.